apps/ubuntu_bootstrap/lib/app/installation_step.dart

```diff
@@ -41,6 +43,8 @@ enum InstallationStep with RouteName {
   refresh(RefreshPage.new, allowedToHide: true),
   tryOrInstall(TryOrInstallPage.new, discreteStep: false, allowedToHide: true),
   autoinstall(AutoinstallPage.new, allowedToHide: true),
+  autoinstallLandscapeDomain(LandscapeDomainPage.new),
+  autoinstallLandscapeQR(LandscapePage.new, required: true),
```

Comment:
Can we make the naming consistent? (the file names should be adjusted correspondigly)
```suggestion
  autoinstallLandscapeDomain(AutoinstallLandscapeDomainPage.new),
  autoinstallLandscapeQr(AutoinstallLandscapeQRPage.new, required: true),
```